testing/test_runner.py

```diff
@@ -537,33 +550,33 @@ def pytest_runtest_teardown(item):
     result.stdout.fnmatch_lines(["*2 passed*"])
 
 
-def test_outcomeexception_exceptionattributes():
+def test_outcomeexception_exceptionattributes() -> None:
     outcome = outcomes.OutcomeException("test")
     assert outcome.args[0] == outcome.msg
 
 
-def test_outcomeexception_passes_except_Exception():
+def test_outcomeexception_passes_except_Exception() -> None:
     with pytest.raises(outcomes.OutcomeException):
         try:
             raise outcomes.OutcomeException("test")
         except Exception:
             pass
 
 
-def test_pytest_exit():
-    with pytest.raises(pytest.exit.Exception) as excinfo:
+def test_pytest_exit() -> None:
+    with pytest.raises(Exit) as excinfo:
         pytest.exit("hello")
-    assert excinfo.errisinstance(pytest.exit.Exception)
+    assert excinfo.errisinstance(Exit)
 
 
-def test_pytest_fail():
-    with pytest.raises(pytest.fail.Exception) as excinfo:
+def test_pytest_fail() -> None:
+    with pytest.raises(Failed) as excinfo:
```

Comment:
Maybe some extra check like `assert isinstance(Failed, pytest.fail.Exception)` might make sense (ensuring that this is still the case really)?

Reply:
Sure, added.